Source/m_menu.c

```diff
@@ -134,7 +134,7 @@ boolean menuactive;    // The menus are up
 #define SKULLXOFF  -32
 #define LINEHEIGHT  16
 
-#define M_SPC        8
+#define M_SPC        9
```

Comment:
The height of the mini-thermo patches is 9, so I increased the `M_SPC`. Now we need to rearrange a few menus again, but I like the 9px spacing more. @fabiangreffrath what do you think?

Reply:
@fabiangreffrath  Should I rearrange the "General" and "Key Bindings" menus because of the 9px spacing? Also I think we can remove "Flashing HOM indicator" and "Maximum number of player corpses" and maybe other items from "General".

Reply:
Okay. If we introduce a wider line spacing anyway, how about condensing the Compatibility menu as well, then?